modules/javafx.controls/src/test/java/test/javafx/scene/control/ComboBoxTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2011, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2011, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
`2022, 2023,`

Reply:
this is one is correct, though? `2011, 2023,` should be fine?